README.md

```diff
@@ -109,43 +109,82 @@ If an alias was in place for the images directory, i.e.
 ```
 Then the svg can be inlined with: `<img inline src="~img/icons.svg">`. This method would require the use of **loaders** on your templates as shown above in point 2.
 
-#### Setting `runPreEmit` option
-If you aren't using **loaders** to resolve file locations, and would prefer to reference image paths relative to the **root** of your project (where your `package.json` file resides) then set the plugins `runPreEmit` config option to `true`:
```

Comment:
None of the original documentation has been removed. It's just looks so when comparing the previous revisions with the new documentation additions.